vraptor-core/src/main/java/br/com/caelum/vraptor/core/UTF8Control.java

```diff
@@ -0,0 +1,89 @@
+/*
+  https://gist.github.com/Hasacz89/d93955ec91afc73a06e3
+ */
+
+package br.com.caelum.vraptor.core;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.security.AccessController;
+import java.security.PrivilegedActionException;
+import java.security.PrivilegedExceptionAction;
+import java.util.Locale;
+import java.util.PropertyResourceBundle;
+import java.util.ResourceBundle;
+
+public class UTF8Control extends ResourceBundle.Control {
```

Comment:
Hi Turini,
I'm new in Java EE, so I don't know about CDI, I need to learn about it.
Thanks for review.